dev/ci/ci-elpi.sh

```diff
@@ -10,7 +10,9 @@ git_download elpi
 if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
 
 ( cd "${CI_BUILD_DIR}/elpi"
+  rm -f dune-workspace # force composed build
```

Comment:
```suggestion
  export DUNE_build_FLAGS="--root ."
```
without this I get `Error: Don't know how to build theories`
with it you shouldn't need to change the artifact handling